src/main/java/seedu/address/storage/JsonAdaptedPassenger.java

```diff
@@ -54,13 +54,13 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
     }
 
     /**
-     * Converts a given {@code Person} into this class for Jackson use.
+     * Converts a given {@code Passenger} into this class for Jackson use.
      */
-    public JsonAdaptedPerson(Person source) {
+    public JsonAdaptedPassenger(Passenger source) {
         name = source.getName().fullName;
         phone = source.getPhone().value;
         address = source.getAddress().value;
-        tripDay = source.getTripDay().value.toString();
```

Comment:
Is the toString method call required? if this Pr is just for renaming was removing the call a mistake?